<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegiWindow</class>
 <widget class="QMainWindow" name="RegiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-image: url(:/R-C.gif);
	background:rgba(255,255,255,100%);
	border-radius:20px;
}
#centralwidget{
	background:rgba(255,255,255,100%);
	border-radius:20px;
	background-color: rgb(140, 140, 140);
}
#GIFLabel{
	border-radius:20px;
background:rgb(25,25,25,80%)
}
#Frame{
	border-radius:15px;
	background:rgba(50,50,50,80%)
}
#Frame_gif{
	border-radius:20px;
	background:transparent;	
}

QPushButton{
	background:#1e1d1d;
	border-radius:8px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)
}

#pushButton{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)

}
#pushButton:hover{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}
#pushButton:pressed {
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(255,255,255); 
	background:rgba(20,20,20,90%)
}
QLineEdit{
	border-radius:15px;
	background:#1e1d1d;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:rgba(20,20,20,90%);
	color: rgb(200,200,200); 
	padding-left: 15px;
}
QLineEdit:hover
{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}
#password_find{
	color: rgb(160,160,160); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:transparent;	
}
#password_find:hover{
	color: rgb(255,255,255); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;
}
#password_find:pressed{
	color: rgb(160,160,160); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;
}
#sign{
	color: rgb(160,160,160); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:transparent;	
}
#sign:hover{
	color: rgb(255,255,255); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;	
}
#sign:pressed{
	color: rgb(160,160,160); 
	font-size: 24px ;
	font-family: Microsoft YaHei ;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>308</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Passwd</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>376</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tel</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>43</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
